[core] Fix doc for User-Spawned Process (#49351)
ray-project/ray: doc/source/ray-core/user-spawn-processes.rst

User-Spawned Process Killed on Worker Exit
------------------------------------------

> In the following example, we use Ray Actor to spawn a user process. The user process is a sleep process.

Reviewer: ```suggestion
The following example uses a Ray Actor to spawn a user process. The user process is a sleep process.
```
